pom.xml

```diff
@@ -39,6 +39,13 @@
             <version>${webdrivermanager.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Remove this dependency please. To avoid issue with missed dependency try to change the scope of the dependency

Reply:
Fixed